beaglecli

```diff
@@ -75,6 +79,7 @@ Usage:
   beaglecli files patch <file_id> [--file-path=<file_path>] [--file-type=<file_type>] [--file-group=<file_group_id>] [--metadata=<metadata>]... [--size=<size>]
   beaglecli files list [--page-size=<page_size>] [--path=<path>]... [--metadata=<metadata>]... [--file-group=<file_group>]... [--file-name=<file_name>]... [--filename-regex=<filename_regex>] [--file-type=<file_type>]... [--all]... [--packaged]... [--force]...
   beaglecli files delete --file-id=<file_id>...
+  beaglecli pipeline update-cache <cache_path>
```

Comment:
I wouldn't mind trying the feature out on an example though. Is there a good cache path I could test it out on?